devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -23,6 +23,9 @@ metadata {
 		capability "Temperature Alarm"
 
 		fingerprint mfr:"010F", prod:"0C02", model:"1002"
+		fingerprint mfr:"010F", prod:"0C02", model:"4002"
```

Comment:
Is there any benefit of including some specific models vs the single fingerprint with must mfg/prod?

Reply:
lets us know which models we've specifically certified, I suppose.

Reply:
The other benefit I see is making sure we don't possibly hit edge cases with existing devices. I feel like if we remove the existing fingerprints then we need to re-test all devices that we removed. Though, maybe we should anyway.